crates/runtime/src/helpers.c

```diff
@@ -73,7 +77,7 @@ __declspec(dllexport)
   // `asm` statement.
 __attribute__((weak, noinline))
 #endif
-void __jit_debug_register_code() {
+void VERSIONED_SYMBOL(__jit_debug_register_code)() {
```

Comment:
Oh this is actually the one symbol which needs to not be versioned since IIRC gdb sets a breakpoint on this or something like that. That's ok though due to the `weak` linkage which should enable this to exist with other definitions ok.

Reply:
Makes sense, removed, thanks!